README.md

```diff
@@ -26,16 +26,15 @@ See the wiki for detailed documentation.
 Ensure that docker is up to date and initialized.
 Spin up mysql locally and validate that it is working:
 ```
-./docker/run-mysql.sh start
+./docker/run-mysql.sh
```

Comment:
the README in develop is the correct one. And we should as the same steps for opendj